bin/ci.zodiac-publish-build

```diff
@@ -13,8 +13,12 @@ git submodule update
 
 cd zodiac-cli
 
-export PATH="../bin:$PATH"
+# Need to hard code these here. When we swicth compilers, these need to be updated.
+export GHC_PATH=/usr/local/ghc-7.10.2/bin
+export CABAL_PATH=/usr/local/cabal-8.0.1/bin
 
-./mafia update
-./mafia build ${CABAL_CONFIG_ARGS}
-./mafia testci ${CABAL_CONFIG_ARGS}
+export PATH="../bin:$GHC_PATH:$CABAL_PATH:$PATH"
+
+mafia update
+mafia build ${CABAL_CONFIG_ARGS}
+mafia testci ${CABAL_CONFIG_ARGS}
```

Comment:
@olorin Are you ok with this? This is the only thing I could find that worked. Using `ghc-path` and `cabal-path` don't work in the case because this script doesn't get the right environment variables from something earlier in the chain.